app-i18n/ibus-anthy/ibus-anthy-1.5.9.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit autotools eutils gnome2-utils python-single-r1
+
+DESCRIPTION="Japanese input method Anthy IMEngine for IBus Framework"
+HOMEPAGE="https://github.com/ibus/ibus/wiki"
+SRC_URI="https://github.com/ibus/ibus-anthy/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="nls"
+
+RDEPEND="${PYTHON_DEPS}
+	app-i18n/anthy
+	app-i18n/ibus[introspection]
+	nls? ( virtual/libintl:0= )"
+
+DEPEND="${RDEPEND}
+	dev-libs/gobject-introspection
+	dev-util/intltool
+	nls? ( sys-devel/gettext )
+	virtual/pkgconfig"
+
+src_prepare() {
+	default
+	eautoreconf
+	>py-compile #397497
+}
+
+src_configure() {
+	econf \
+		--libexecdir=/usr/libexec \
```

Comment:
Do you actually need this? It looks like the default to me. Besides, wouldn't harm to prepend `"${EPREFIX}"` to the path.